test_files/namespaced/export_namespace.ts

```diff
@@ -1,3 +1,8 @@
+// TODO(#132): namespaces currently don't emit properly.
+// This workaround at least makes them compile.
+// @ts-ignore
+exports = {};
+
 // tslint:disable:no-namespace
 
 export namespace valueNamespace {
```

Comment:
I think that a nicer way to handle this case is:

```typescript
namespace valueNamespace {
  export class ValueClass {}
}
namespace typeNamespace {
  export interface Interface {}
}

export {valueNamespace, typeNamespace)
```

which I think just happens to have an emit where the `goog.module` exports are in the expected form.

That said, since namespaces are against our style guide, it might not be as important.

Reply:
I left this one alone, and changed the other one.